specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/ManagedDatabaseMoveOperations.json

```diff
@@ -0,0 +1,293 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/locations/{locationName}/managedDatabaseMoveOperationResults": {
+      "get": {
+        "tags": [
+          "ManagedDatabaseMoveOperations"
+        ],
+        "description": "Gets managed database move operations.",
+        "operationId": "ManagedDatabaseMoveOperations_ListByLocation",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "name": "locationName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "onlyLatestPerDatabase",
```

Comment:
>onlyLatestPerDatabase [](http://example.com/codeflow?start=21&length=21)

have you considered using one of the standard parameters : [https://github.com/microsoft/api-guidelines/blob/62502e094f8a3e16586ab039d7bbe1021ec7804d/azure/Guidelines.md#query-options](https://github.com/microsoft/api-guidelines/blob/62502e094f8a3e16586ab039d7bbe1021ec7804d/azure/Guidelines.md#query-options)


Also consider if you want to pass in this value in body rather than query

Reply:
@raosuhas Thanks for pointing this one out, and I'd say that the optional "filter" argument is meant to provide flexibility via standard query options. On the other hand, "onlyLatestPerDatabase" is a shortcut meant to provide the result we expect to be used most often (i.e. by Portal).